ROMFS/px4fmu_common/init.d/rc.autostart

```diff
@@ -0,0 +1,318 @@
+#
```

Comment:
You're re-comitting it with this patch set, and you have your modification in it.